src/app/Main/Body/QueryPage/Results/Charts/BarChartResults.component.tsx

```diff
@@ -9,6 +9,7 @@ export function BarChartResultsComponent(props: {
     getPngDownloadUriRef: MutableRefObject<(() => Promise<string>) | null>;
     xAxisLabel: string;
     yAxisLabel: string;
+    xAxisMax: number | undefined;

```

Comment:
I recommend to use the short form `xAxisMax?: number` for optional props.

Reply:
Done.